app/views/ReportView/components/AnalystComments/index.tsx

```diff
@@ -99,40 +99,45 @@ const AnalystComments = ({
   };
 
   const handleEditorClose = useCallback(async (editedComments?: string) => {
-    setIsEditorOpen(false);
-    if (editedComments) {
-      const sanitizedText = sanitizeHtml(editedComments, {
-        allowedAttributes: {
-          a: ['href', 'target', 'rel'],
-        },
-        transformTags: {
-          a: (_tagName, attribs) => ({
-            tagName: 'a',
-            attribs: {
-              href: attribs.href,
-              target: '_blankk',
-              rel: 'noopener noreferrer',
-            },
-          }),
-        },
-      });
-      const commentCall = api.put(
-        `/reports/${report.ident}/summary/analyst-comments`,
-        { comments: sanitizedText },
-      );
-
-      if (isSigned) {
-        showConfirmDialog(commentCall);
-      } else {
-        // If signed, the dialog that opens up will refesh the page instead
-        const commentsResp = await commentCall.request();
-        setComments(sanitizeHtml(commentsResp?.comments, {
-          allowedSchemes: [],
+    try {
+      if (editedComments) {
+        const sanitizedText = sanitizeHtml(editedComments, {
           allowedAttributes: {
-            '*': ['style'],
+            a: ['href', 'target', 'rel'],
           },
-        }));
+          transformTags: {
+            a: (_tagName, attribs) => ({
+              tagName: 'a',
+              attribs: {
+                href: attribs.href,
+                target: '_blankk',
```

Comment:
I realise this was there already, but is it definitely not a typo?

Reply:
` interface Tag { tagName: string; attribs: Attributes; text?: string | undefined; }`

is a legit parameter

Reply:
oh sorry you meant the blankk

Reply:
that's probably a typo then